python/taichi/examples/simulation/laplace_equation.py

```diff
@@ -274,7 +274,7 @@ def processGuiEvent(_gui):
                         else:
                             dipoles[i].m += 0.001 * int((dipoles[i].m >= 0.0) -
                                                         (dipoles[i].m <= 0.0))
-        fade = -ti.abs(fade)  # fade out arrow filed
+        fade = -abs(fade)  # fade out arrow filed
```

Comment:
```suggestion
        fade = -abs(fade)  # fade out arrow field
```
I think you mean `field`, not `filed`?

Reply:
@bismarckkk Please resolve this conversation?